dashboard/app/views/levels/_dialogHelper.js

```diff
@@ -0,0 +1,57 @@
+/* global $ skipSound preTitle lastServerResponse Dialog */
+
+var dialogType = null;
+var adjustedScroll = false;
+
+$(function () {
+  if (preTitle != "") {
+    window.setTimeout(function () {
+      showDialog("pre");
+    }, 1000);
+  }
+});
```

Comment:
Interesting. My preference would be to use the first form to make this more explicit.